src/style.css

```diff
@@ -1 +1,44 @@
-/* add styles here */
+* {
```

Comment:
Using the universal selector `*` to reset margins can lead to performance issues. Consider using more specific selectors to achieve the desired effect.